// Copyright (c) 2009 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for bookmarks.

// Update proto_value_conversions{.h,.cc,_unittest.cc} if you change
// any fields in this file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;
option retain_unknown_fields = true;

package sync_pb;

import "sync.proto";

// Properties of bookmark sync objects.
message BookmarkSpecifics {
  optional string url = 1;
  optional bytes favicon = 2;
}

extend EntitySpecifics {
  optional BookmarkSpecifics bookmark = 32904;
}
